app/src/stores/userStore.ts

```diff
@@ -33,11 +26,6 @@ interface UserState {
 }
 
 const useUserStore = create<UserState>((set, _get) => ({
-  passportNumber: DEFAULT_PNUMBER ?? '',
```

Comment:
@transphorm are those `DEFAULT_*` consts something that need to be moved to the SDK or empty strings as default is fine?